docs/environment_variables.md

```diff
@@ -79,3 +79,24 @@ SERVER_TRAFFIC_SECRET_0 XXXX
 CLIENT_HANDSHAKE_TRAFFIC_SECRET XXXX
 CLIENT_TRAFFIC_SECRET_0 XXXX
 ```
+
+`SSL_CERT_FILE` `REQUESTS_CA_BUNDLE` `CURL_CA_BUNDLE`
+-----------
+
+Valid values: a filename
+
+If one of these environment variables is set and `trust_env=True` and `verify=True` or `verify is None`, `verify` will be set by respectively `SSL_CERT_FILE`, `REQUESTS_CA_BUNDLE`, `CURL_CA_BUNDLE` environments as file.
+
+Example:
+
+```python
+# test_script.py
+
+import httpx
+client = httpx.Client(verify=None, trust_env=True)
```

Comment:
I think we should just document this as `client = httpx.Client()`, since we'll anyways default to supporting `SSL_CERT_FILE` *unless* verify/trust_env has been overridden.